gradle.properties

```diff
@@ -39,10 +39,12 @@ commonsLang3Version=3.12.0
 commonsMath3Version=3.6.1
 javaPoetVersion=1.12.0
 log4jVersion=2.17.2
-guavaVersion=30.1-jre
+guavaVersion=31.0.1-jre
```

Comment:
It's possible we should increment the minor version of the Record Layer prior to merging this and warn people about the guava version bump

Reply:
Upgraded as discussed to `31.1-jre`